server/requirements.txt

```diff
@@ -5,4 +5,4 @@ python3-saml==1.2.6
 lxml==3.8.0
 feedgen==0.9.0
 
--e git+git://github.com/superdesk/superdesk-core.git@develop#egg=Superdesk-Core
+-e git+git://github.com/superdesk/superdesk-core.git@master#egg=Superdesk-Core
```

Comment:
we can use `superdesk-core>=1.33.3,<1.34` actually

Reply:
done

Reply:
`>=1.33.3,<1.34` won't work in `requirements.txt` I set `1.33.3` and we'll update when necessary.

Reply:
OK got it it's not a branch on the repos, but a version on pypi, fixing it